### PR TITLE
Support multiple keys

### PR DESCRIPTION
**Pull request by aermakov-zalando**
We need this to support key rotation.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ echo 'ssh_access_granting_service_url: "'$GRANTING_SERVICE_URL'"' > /etc/ssh-acc
 echo 'allowed_remote_networks: ['$ALLOWED_REMOTE_NETWORKS']' >> /etc/ssh-access-granting-service.yaml
 
 echo 'Writing SSH public key..'
-echo 'command="grant-ssh-access-forced-command.py" '$GRANTING_SERVICE_SSH_KEY > ~granting-service/.ssh/authorized_keys
+echo "$GRANTING_SERVICE_SSH_KEY" | sed 's/^/command="grant-ssh-access-forced-command.py" /' > ~granting-service/.ssh/authorized_keys
```
Comment (tuxlife) [inline run.sh:17]: `GRANTING_SERVICE_SSH_KEY` or `GRANTING_SERVICE_SSH_KEYS`?

File: README.rst [hunk 2]
```diff
@@ -12,8 +12,8 @@ The Docker image expects the following environment variables to be set:
 
 ``ALLOWED_REMOTE_NETWORKS``
     List of IP networks (CIDR) the bastion host is allowed to access.
-``GRANTING_SERVICE_SSH_KEY``
-    The public SSH key for the "granting-service" user.
+``GRANTING_SERVICE_SSH_KEYS``
```
Comment (tuxlife) [inline README.rst]: You changed the Variable only in the readme

Comment (aermakov-zalando) [reply README.rst]: Oops, fixed.

**Comment by tuxlife** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**